### PR TITLE
Wait actual end of CompileTools.runAction

### PR DESCRIPTION
**Pull request by sebjulliand**
### Changes
Since `CompileTools.runAction` now uses task, it returns right after the task is created and started.
This PR makes it wait for the actual end of the action before returning and also lets the `code-for-ibmi.runAction` command returns the result of the action as a `boolean`.

### Checklist

* [x] have tested my change
* [x] Remove any/all `console.log`s I added

### CODE REVIEW

File: src/languages/clle/clRunner.ts [hunk 1]
```diff
@@ -25,14 +23,16 @@ export function initialise(context: ExtensionContext) {
               );
             }
             
-            CompileTools.runAction(instance, editor.document.uri, {
+            return await CompileTools.runAction(instance, editor.document.uri, {
```
Comment (worksofliam) [inline src/languages/clle/clRunner.ts:26]: Is there a reason why the `return` statements were added here? Generally don't want this API to used like that, and actually will eventually be moved into the CL extension.

**Review by worksofliam — DISMISSED** [review]
Works as expected, just check out my comment about the CL runner above.

File: src/languages/clle/clRunner.ts (hunk 1 above)
Comment (sebjulliand) [reply src/languages/clle/clRunner.ts:26]: Mainly for consistency and in the event it gets called by another extension.
But if it's not intended to be used externally or even this way, I'll be happy to roll it back 😉

Comment (worksofliam) [reply src/languages/clle/clRunner.ts:26]: I think I'd prefer it if you did! Don't want to let people get comfy with it and then have me remove it down the line!~

**Comment by sebjulliand** [comment]
@worksofliam there you go!

**Review by worksofliam — APPROVED** [review]
A nice functional change. Thanks @sebjulliand!

**Pull request merged**